hazelcast-ra/hazelcast-jca/src/test/resources/ra.xml

```diff
@@ -23,7 +23,7 @@
 
     <vendor-name>Hazelcast.com</vendor-name>
     <eis-type>Hazelcast</eis-type>
-    <resourceadapter-version>3.2-RC3-SNAPSHOT</resourceadapter-version>
+    <resourceadapter-version>3.6-SNAPSHOT</resourceadapter-version>
```

Comment:
yes, it is updated https://github.com/hazelcast/DevOps/blob/master/build-tool/change-version.sh#L16